packages/components/src/components/Table/TableRow.tsx

```diff
@@ -1,30 +1,36 @@
 import { ReactNode } from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
-import { Elevation, mapElevationToBorder, spacingsPx } from '@trezor/theme';
+import { Elevation, mapElevationToBorder } from '@trezor/theme';
 
 import { useElevation } from '../ElevationContext/ElevationContext';
 
-export const Row = styled.tr<{ $elevation: Elevation }>`
-    display: grid;
-    grid-auto-flow: column;
-    grid-auto-columns: 1fr;
-    align-items: center;
-    margin: 0 ${spacingsPx.xl};
-    padding: ${spacingsPx.lg} 0;
-    border-bottom: 1px solid ${mapElevationToBorder};
-
-    &:last-child {
-        border-bottom: none;
+export const Row = styled.tr<{ $elevation: Elevation; $isCollapsed: boolean }>`
+    border-top: 1px solid ${mapElevationToBorder};
+
+    &:first-child {
+        border-top: 0;
     }
+
+    ${({ $isCollapsed }) =>
+        $isCollapsed &&
+        css`
+            visibility: collapse;
```

Comment:
AssetLogo's visibility collides with this and makes icons visible from `Zero-balance tokens` section even though it is not open
<img width="526" alt="Screenshot 2024-09-14 at 7 31 33" src="https://github.com/user-attachments/assets/3554820a-e0ce-4484-8cb4-12f207b4bdc8">

Reply:
Weird! Doesn't happen on my side. Will investigate.

Reply:
Fixed.